Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -180,14 +180,21 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         toolbar.layer.shadowRadius = 2.0
         toolbar.layer.shadowOpacity = 0.25
 
-        let settingsItem = UIBarButtonItem(title: "\u{2699}", style: .Plain, target: self, action: "SELdidClickSettingsItem")
+        let settingsItem = UIBarButtonItem(image: UIImage(named: "search"), style: .Plain, target: self, action: "SELdidClickSettingsItem")
         settingsItem.accessibilityLabel = NSLocalizedString("Settings", comment: "Accessibility label for the Settings button in the Tab Tray.")
         let signinItem = UIBarButtonItem(title: NSLocalizedString("Sign in", comment: "Button that leads to Sign in section of the Settings sheet."),
             style: .Plain, target: self, action: "SELdidClickDone")
         signinItem.enabled = false
-        // TODO: Vertically center the add button.  Right now, it's too high in the containing bar.
-        let addTabItem = UIBarButtonItem(barButtonSystemItem: .Add, target: self, action: "SELdidClickAddTab")
+
+        let addTabItem = UIBarButtonItem(image: UIImage(named: "search"), style: .Plain, target: self, action: "SELdidClickAddTab")
+        
+        let topInset: CGFloat = StatusBarHeight/2;
```

Comment:
Value 10.0 which obtained through trial and error is not random it changes with the change in StatusBarHeight. Vertical height adjustment had to be done for both barButtomSystemItem and Image and not for unicode characters. Currently images are set to a random image(search) once assets for these two are loaded they will be changed.